_data/security.yml

```diff
@@ -1,4 +1,14 @@
 websites:
+  - name: Abode
+    url: https://goabode.com
+    img: abode.png
+    tfa:
+      - totp
+    doc: https://help.goabode.com/hc/en-us/articles/360022414751
+    regions:
+      - ca
+      - us
```

Comment:
```suggestion
      - us
      - gb
```
Also available in the UK: https://goabode.co.uk/

Reply:
I think the "Security" category is more targeted at _online security_, not _physical security_, so IMHO it can stay in the "IoT" category.

Reply:
It is in security at the moment, I requested it be moved to IoT :grinning:

Reply:
Did I mention that I can't read? :sweat_smile: